.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,102 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for your time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Provide a clear and concise description of what the bug is.
+      placeholder: Tell us a description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: step-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Provide details of how to reproduce the bug.
+      placeholder: ex. 1. Go to '...'
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What did you expect to happen?
+      placeholder: ex. I expected '...' to happen
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: What did actually happen?
+      placeholder: ex. Instead '...' happened
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: Operating system
+      description: Which operating system are you using?
+      options:
+        - Windows
+        - macOS
+        - Linux
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: python-version
+    attributes:
+      label: Python version
+      description: Which version of Python are you using?
+      options:
+        - Python 3.12
+        - Python 3.11
+        - Python 3.10
+        - Python 3.9
+        - Python 3.8
+        - Python 3.7
+        - Python 3.6
+        - Python 3.5
+      default: 1
+    validations:
+      required: true
+  - type: dropdown
+    id: mytimer-version
+    attributes:
+      label: MyTimer version
+      description: Which version of MyTimer are you using?
+      options:
+        - MyTimer 1.1
+        - MyTimer 1.0
+        - MyTimer 0.9
+        - MyTimer 0.7
```

Comment:
MyTimer 0.8

Reply:
fixed in [6917646](https://github.com/sepandhaghighi/mytimer/pull/80/commits/6917646dcf94b3d4d9cc059949bc17adf3ab95ef).